packages/@aws-cdk/aws-events-targets/lib/firehose-stream.ts

```diff
@@ -0,0 +1,47 @@
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import * as firehose from '@aws-cdk/aws-kinesisfirehose';
+import { singletonEventRole } from './util';
+
+/**
+ * Customize the Firehose Stream Event Target
+ */
+export interface FirehoseStreamProps {
+  /**
+   * The message to send to the stream.
+   *
+   * Must be a valid JSON text passed to the target stream.
+   *
+   * @default - the entire CloudWatch event
```

Comment:
```suggestion
   * @default - the entire EventBridge event
```

Reply:
Happy to change it, had left it since this is what all the Kinesis target called it

https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-events-targets/lib/kinesis-stream.ts#L22

Reply:
we've been trying to do this name change and haven't quite updated all the references. It would be good not to proliferate if possible - see #8020